docs/content/tutorial/writing-your-first-asset.mdx

```diff
@@ -22,45 +22,65 @@ By the end of this section, you will:
 
 In Dagster, the main way to create data pipelines is by writing assets. By the end of this tutorial section, you'll have written your first asset.
 
-The scaffolded Dagster project has a directory called `tutorial-project`. If you named your project differently when running the Dagster CLI, the directory will be named after that.
+The scaffolded Dagster project has a directory called `tutorial`. If you named your project differently when running the Dagster CLI, the directory will be named after that.
 
-The `tutorial-project` directory has a file named `assets.py`. This is where you'll put your first data pipeline.
+The `tutorial` directory has a file named `assets.py`. This is where you'll put your first data pipeline.
 
 ### Ingesting data
 
 To get started, you will fetch data from the Hacker News API. Copy and paste the following code into `assets.py`:
 
 ```python
+import json
+import os
+
 import requests
 
 newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
 top_new_story_ids = requests.get(newstories_url).json()[:100]
+
+os.makedirs("data", exist_ok=True)
+with open("data/topstory_ids.json", "w") as f:
+    f.write(json.dumps((top_new_story_ids)))
 ```
 
-This code creates a list of integers representing the IDs for the current top stories on Hacker News.
+This code creates a list of integers representing the IDs for the current top stories on Hacker News and stores them in a file called `data/topstory_ids.json`.
 
 Next, you will work towards making this code into a software-defined asset. The first step is turning it into a function:
 
 ```python
+import json
+import os
+
 import requests
 
-def topstory_ids(): # turn it into a function
+
+def topstory_ids() -> None: # turn it into a function
     newstories_url = "https://hacker-news.firebaseio.com/v0/topstories.json"
     top_new_story_ids = requests.get(newstories_url).json()[:100]
-    return top_new_story_ids # return the data
+
+    os.makedirs("data", exist_ok=True)
+    with open("data/topstory_ids.json", "w") as f:
+        f.write(json.dumps((top_new_story_ids)))
```

Comment:
Are all these parentheses needed?

Reply:
Also, I think you can just do `json.dump` to avoid needing to invoke `f.write`. Same on the read side too.

Reply:
🤦